features/admin_clicks_cat_to_see_details.feature

```diff
@@ -0,0 +1,89 @@
+
+    And the following users exists
+      | email           | admin | member |
+      | user1@mutts.com |       | true   |
+      | user2@mutts.com |       | true   |
+      | user3@mutts.com |       | true   |
+      | user4@mutts.com |       | true   |
+      | user5@mutts.com |       | true   |
+      | admin@shf.se    | true  | false  |
+
+    Given the following payments exist
+      | user_email      | start_date | expire_date | payment_type | status | hips_id |
+      | user2@mutts.com | 2017-10-1  | 2017-12-31  | member_fee   | betald | none    |
+      | user3@mutts.com | 2017-10-1  | 2017-12-31  | member_fee   | betald | none    |
+
+
+    And the following business categories exist
+      | name         |
+      | Groomer      |
+      | Psychologist |
+      | Trainer      |
+      | Rehab        |
+      | Walker       |
+      | JustForFun   |
+
+    And the following applications exist:
+      | user_email      | company_number | categories              | state    |
+      | user1@mutts.com | 5560360793     | Groomer, JustForFun     | accepted |
+      | user2@mutts.com | 2120000142     | Groomer, Trainer, Rehab | accepted |
+      | user3@mutts.com | 6914762726     | Psychologist, Groomer   | accepted |
+      | user4@mutts.com | 6613265393     | Groomer                 | accepted |
+      | user5@mutts.com | 2120000142     | Psychologist            | accepted |
+
+    And the following payments exist
+      | user_email   | start_date | expire_date | payment_type | status | hips_id | company_number |
+      | admin@shf.se | 2017-01-01 | 2017-12-31  | branding_fee | betald | none    | 5560360793     |
+      | admin@shf.se | 2017-01-01 | 2017-12-31  | branding_fee | betald | none    | 2120000142     |
+      | admin@shf.se | 2017-01-01 | 2017-12-31  | branding_fee | betald | none    | 6613265393     |
+      | admin@shf.se | 2017-01-01 | 2017-12-31  | branding_fee | betald | none    | 6222279082     |
+      | admin@shf.se | 2017-01-01 | 2017-12-31  | branding_fee | betald | none    | 8025085252     |
+      | admin@shf.se | 2017-01-01 | 2017-12-31  | branding_fee | betald | none    | 6914762726     |
+      | admin@shf.se | 2017-01-01 | 2017-12-31  | branding_fee | betald | none    | 7661057765     |
+      | admin@shf.se | 2017-01-01 | 2017-12-31  | branding_fee | betald | none    | 7736362901     |
+
+  @focus
+  Scenario: Show company details to admin
+    Given I am logged in as "admin@shf.se"
+    And I am the page for company number "6914762726"
+    Then I should see "Groomer"
+    And I should see "Psychologist"
+    When I click on "Groomer"
+    Then I should see "Groomer"
+    And I should see "PsycGroomerCo"
+    And I should see "GroomerNext"
+    And I should see "kingGroomer"
+    And I should see "Groomer1"
+    And I should not see "stockholmCo"
+    And I should not see "Company7"
+    And I should not see "Company4"
+    And I should not see "Company5"
```

Comment:
We should also test the reverse cases - that is, a visitor (not logged in), and a member, _can not_ see the business category link when they view a company.

You can use the step at line 30 in `features/step_definitions/assertion_steps.rb` to check that:

`Then I should not see <link label> link`